Formula/calicoctl.rb

```diff
@@ -0,0 +1,29 @@
+class Calicoctl < Formula
+  desc "Calico CLI tool"
+  homepage "https://www.projectcalico.org"
+  url "https://github.com/projectcalico/calicoctl/archive/v1.6.1.tar.gz"
+  sha256 "e6ebda9cd2029057f79549cc7143e5c056c530c96ed93d7387f52a3a33137cef"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    ENV["CALICOCTL_VERSION"] = "v1.6.1"
+    ENV["CALICOCTL_GIT_REVISION"] = "1724e011"
```

Comment:
What happens if this is excluded?

Reply:
Its build metadata that needs to be passed to the go build. Otherwise the application would return no version.

```sh
➜  calicoctl --version
calicoctl version v1.6.1, build 1724e011
```

Without those variables
```sh
➜  calicoctl --version
calicoctl version , build 
```

Reply:
We should probably just build from a git checkout then.

Reply:
Agree completely. I don't know why I carried on and didn't switch to `git` before.
It is now switched to using `git checkout` and removed the need for the manual environment variables.